### DISCUSSION TITLE
Polls: Allow more options

### DISCUSSION THREAD

**Discussion by nschloe** [community/community#15841] [Discussions]
I just wanted to move a poll that up until now I had on Google Docs to the new GitHub poll functionality. It's in [meshio](https://github.com/nschloe/meshio/), a tool for working with meshes in different formats, and [it's asking about which formats are actually used](https://docs.google.com/forms/d/e/1FAIpQLSeXXVt2fmJ0S8NuN6TGlYXn17NBf68Qnv9Og2K5_kHfPI6fYw/viewform).

However, I noticed that the maximum number of answers is limited to 8, so I cannot use it for this purpose. I'd be happy if this limit could be extended. (There are about 30 mesh formats now.)

**Comment by endolith** [comment]
Yeah these polls are pretty useless.  Only FPTP voting method, and only 8 options.  It should allow for Approval Voting at least (checkboxes instead of radio buttons), and Condorcet ranking or STAR Voting would be even better.
